cfgov/unprocessed/apps/financial-well-being/css/main.less

```diff
@@ -0,0 +1,183 @@
+.save-links {
+  padding-bottom: 15px;
+  border-bottom: 1px solid #b4b5b6;
+  margin: 30px 0;
+  text-align: right;
+}
+.spectrum {
+  padding-top: 120px;
+  margin-top: 30px;
+  position: relative;
+  max-width: 770px;
+}
+.score-box {
+  height: 150px;
+  border-color: #101820;
+  border-style: solid;
+  position: absolute;
+  top: 0;
+}
+.score-box__left {
+  padding-left: 10px;
+  border-width: 0 0 0 1px;
+}
+.score-box__right {
+  padding-right: 10px;
+  border-width: 0 1px 0 0;
+  text-align: right;
+}
+.score-box__avg {
+  height: 70px;
+  top: 80px;
+}
+.score-box a {
+  border-bottom-width: 1px;
+}
+.spectrum_x-label {
+  padding-top: 5px;
+  padding-left: 5px;
+  border-left: 1px solid #b4b5b6;
```

Comment:
I'll convert the hardcoded colors to variables later. This is just a verbatim copy to help with reviewing.